### DISCUSSION TITLE
EventName without single quotes (Event navigability)

### DISCUSSION THREAD

**Discussion by Arthurvdv** [StefanMaron/BusinessCentral.LinterCop#296] [New Rule]
![image](https://github.com/StefanMaron/BusinessCentral.LinterCop/assets/44637996/5cd6fbc4-df58-401a-8d39-b2b78209caf4)
[Business Central 2023 wave 1 (BC22) new features: Event navigability](https://yzhums.com/36573/)

When the project has a runtime set to 11.0 (or higher) a rule shows to use EventName without single quotes around them.

**Comment by StefanMaron** [comment]
good one, I did not even know the syntax changed. I guess I should have read the release note a bit more carefully ;)
